tasks/main.yml

```diff
@@ -50,6 +59,14 @@
   when: "{{ freshclam_enabled }}"
   tags: usb
 
+- name: Set cron job for scanning / at 12:00 on a Wednesday
```

Comment:
I've looked into the report, they used clamtk to set scheduled scans, these add the scan to the users crontab.

Since users won't have root permissions, setting the schedule inside of clamtk would mean that systemwide scans (which is what they suggested) would not be run.

For reference this is what clamtk schedules:
```0 12 * * * /usr/bin/clamscan --exclude-dir=/home/yousof/.clamtk/viruses --exclude-dir=smb4k --exclude-dir=/run/user/yousof/gvfs --exclude-dir=/home/yousof/.gvfs --exclude-dir=.thunderbird --exclude-dir=.mozilla-thunderbird --exclude-dir=.evolution --exclude-dir=Mail --exclude-dir=kmail -i -r /home/yousof --log="$HOME/.clamtk/history/$(date +\%b-\%d-\%Y).log" 2>/dev/null # clamtk-scan````

This does a scan of the users home directory not a systemwide scan